src/CONFIG.js

```diff
@@ -60,6 +60,6 @@ export default {
         DEFAULT: '/favicon.png',
         UNREAD: '/favicon-unread.png',
     },
-    CAPTURE_METRICS: lodashGet(Config, 'CAPTURE_METRICS', false),
-    ONYX_METRICS: lodashGet(Config, 'ONYX_METRICS', false),
+    CAPTURE_METRICS: lodashGet(Config, 'CAPTURE_METRICS', 'false') === 'true',
+    ONYX_METRICS: lodashGet(Config, 'ONYX_METRICS', 'false') === 'true',
```

Comment:
**Note:** These changes are not related, but I noticed my test builds were showing the metrics and decided to fix this here.